### PR TITLE
initial commit for social media page

### PR DESCRIPTION
**Pull request by Ark-Arch**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import SocialMediaPage from './SocialMediaManagement/SocialMediaPage'
 
 function App() {
   return (
     <div className="text-left text-sky-600">
-      <div>This is a new project</div>
```
Comment (SwiftEsther) [inline src/App.js:7]: This should be removed.

File: src/Assets/Facebook.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+const Facebook = () => {
+    return (
+        <div className='Facebook'>
```
Comment (SwiftEsther) [inline src/Assets/Facebook.js]: The outer div should be gotten rid of.

File: src/Assets/Instagram.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Instagram = () => {
+    return(
+        <div className="Instagram">
```
Comment (SwiftEsther) [inline src/Assets/Instagram.js]: The outer div should be gotten rid of.

File: src/Assets/Twitter.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+
+const Twitter = () => {
+    return (
+        <div className='Twitter'>
```
Comment (SwiftEsther) [inline src/Assets/Twitter.js]: The outer div should be gotten rid of.

File: src/Assets/Logo.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const Logo = () => {
+    return (
+        <div className='Logo'>
```
Comment (SwiftEsther) [inline src/Assets/Logo.js]: The outer div should be gotten rid of.

**Review by SwiftEsther — CHANGES_REQUESTED** [review]

File: src/Assets/test.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+// {/* <div class="bg-gray-100 pt-2">
```
Comment (SwiftEsther) [inline src/Assets/test.js]: This file should be deleted if it has no purpose

File: src/Components/Callout.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {ReactComponent as WebProgramming} from './images/web-programming.svg'
+
+const Callout = (props) => {
+    const calloutHeading = props.calloutHeading;
```
Comment (SwiftEsther) [inline src/Components/Callout.js]: variable not in use. Remove till it is needed

File: src/Components/Callout.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    const calloutNotes = props.calloutNotes;
```
Comment (SwiftEsther) [inline src/Components/Callout.js]: variable not in use. Remove till it is needed

File: src/Components/FeaturePageElements/LeftHandImage.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const LeftHandImage = ({lhiHeader, lhiParagraph, lhiImage}) => {
```
Comment (SwiftEsther) [inline src/Components/FeaturePageElements/LeftHandImage.js:3]: Fix component styling to match Figma design

File: src/Components/FeaturePageElements/RightHandImage.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const RightHandImage = ({rhiHeader, rhiParagraph, rhiImage}) => {
```
Comment (SwiftEsther) [inline src/Components/FeaturePageElements/RightHandImage.js:3]: Fix component styling to match Figma design

File: src/Components/FeaturePageElements/LeftHandImage.js [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    return (
+        <div className="LeftHandImage">
```
Comment (SwiftEsther) [inline src/Components/FeaturePageElements/LeftHandImage.js]: For consistency, make the className snake-case like the others

File: src/Components/FeaturePageElements/RightHandImage.js [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+        <div className="RightHandImage">
```
Comment (SwiftEsther) [inline src/Components/FeaturePageElements/RightHandImage.js]: For consistency, make the className snake-case like the others

File: tailwind.config.js [hunk 13]
```diff
@@ -4,7 +4,13 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {colors:{
+      'regal-blue': '#1D8BCC'
```
Comment (SwiftEsther) [inline tailwind.config.js]: Does the hex match the name "regal-blue" or should it be changed?

File: tailwind.config.js [hunk 14]
```diff
@@ -4,7 +4,13 @@ module.exports = {
...
+    },
+    backgroundImage:{
+      'lpBackground': "url('./LandingPage/lpBackground.svg')",
```
Comment (SwiftEsther) [inline tailwind.config.js]: Explanatory keys and variable names

File: src/SocialMediaManagement/SocialMediaPage.js [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+    rhiImage: <img src={ContentWriting} alt="Content Writing"/>
+}
+
+const SocialMediaPage = () => {
+    return (
+        <div className="SocialMediaPage">
+            <Header />
+
+            <div>
+                <img src={RectangleSM} alt="rectangle"/>
```
Comment (SwiftEsther) [inline src/SocialMediaManagement/SocialMediaPage.js]: This alt doesn't exactly give any information about the image. Use something that is easy to understand like you have with the rhiImage, lhiImage, and co.

File: src/LandingPage/LandingPage.js [hunk 16]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {ReactComponent as LPImage} from './lpImage.svg';
+
+const LandingPage = () => {
```
Comment (SwiftEsther) [inline src/LandingPage/LandingPage.js:4]: Fix component styling to match Figma design

File: src/LandingPage/LandingPage.js [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+    return(
+        <div className="LandingPage">
```
Comment (SwiftEsther) [inline src/LandingPage/LandingPage.js]: For consistency, make the className snake-case like the others

File: src/Components/Footer.js [hunk 18]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Facebook from "../Assets/Facebook";
+import Twitter from "../Assets/Twitter"
+import Instagram from "../Assets/Instagram"
+import Logo from "../Assets/Logo"
+
+const Footer = () => {
+    return (
+        <div className="Footer">
```
Comment (SwiftEsther) [inline src/Components/Footer.js]: For consistency, make the className snake-case like the others

File: src/Components/Footer.js [hunk 19]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Facebook from "../Assets/Facebook";
+import Twitter from "../Assets/Twitter"
+import Instagram from "../Assets/Instagram"
+import Logo from "../Assets/Logo"
+
+const Footer = () => {
```
Comment (SwiftEsther) [inline src/Components/Footer.js:7]: Fix component styling to match Figma design

File: src/Components/Header.js [hunk 20]
```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import Logo from "../Assets/Logo"
+
+const Header = () => {
```
Comment (SwiftEsther) [inline src/Components/Header.js:5]: Fix component styling to match Figma design

**Review by SwiftEsther — CHANGES_REQUESTED** [review]
General observation: change "class" attributes to "className" on elements

File: tailwind.config.js (hunk 13 above)
Comment (Ark-Arch) [reply tailwind.config.js]: the actual name is "Newport Blue", so i'll change "regal-blue" to "newport-blue"

File: tailwind.config.js (hunk 14 above)
Comment (Ark-Arch) [reply tailwind.config.js]: Do you mean I should re-construct the variable keys and names?

**Review by SwiftEsther — APPROVED** [review]

**Comment by SwiftEsther** [comment]
We will merge this for now. @Ark-Arch following our chat on this commit, kindly delete the LeftHandImage and RightHandImage components in the next commit.

**Pull request merged**